x-pack/packages/kbn-cloud-security-posture-common/utils/helpers.ts

```diff
@@ -48,7 +48,7 @@ export const buildEntityFlyoutPreviewQuery = (field: string, queryValue?: string
       filter: [
         {
           bool: {
-            should: [{ term: { [field]: { value: `${queryValue || ''}` } } }],
+            should: [{ match: { [field]: `${queryValue || ''}` } }],
```

Comment:
for transparency summarising our discussion in DM here as well: the `term` didn't work for wiz vulnerability data because it was missing explicit mapping for `host.name` and `user.name` fields. As a result, it was mapped to `text` which can't query with `term`. `match` doesn't work for us because for values like `host name with spaces`, it would search through all tokens in the name, and will return all results for terms `host`, `name` in different combinations. So far `match_phrase` looks like the safest option, as it should work for both `text` and `keyword` mapping. an alternative would be to leave `term` here and rely on the future fix https://github.com/elastic/security-team/issues/10750

Reply:
upon further discussion we decided to proceed with `term` as it works for misconfig both wiz (once  https://github.com/elastic/security-team/issues/10750  is fixed) and our native and for our native vulnerability.